TypeInjections/Tool/Tool.fs

```diff
@@ -21,31 +21,38 @@ module Tool =
         let commitRepo = newRepo.Lookup<Commit>(hash)
         Commands.Checkout(newRepo, commitRepo) |> ignore
 
-    // based off command line flags, create and provide path for input/output folders
-    let interpretFlag (argvList: string list) =
-        let flag = argvList.Item(0)
-
-        // -p [inputPath] [inputPath] [outputPath]
-        if flag.Contains("-p") then
-            Console.WriteLine("***** local project comparison")
-
-            if argvList.Length <> 4 then
-                failwith "usage: program -p OLD NEW OUTPUTFOLDER"
 
-        // -l [git commit hash] [git commit hash] [OutputPath]
-        elif flag.Contains("-l") then
-            Console.WriteLine("***** git commit")
+    type options =
+        { [<Option('d', "delete", Required = false, HelpText = "use option to remove New and Old folders")>]
+          delete: bool
+          [<Option('l', "local", Required = false, HelpText = "[inputPath] [inputPath] [outputPath]")>]
+          local: seq<string>
+          [<Option('g', "git", Required = false, HelpText = "[old commit] [new commit] [outputPath]")>]
```

Comment:
Why does the `git` command take an `outputPath`?

Reply:
`local`, `git` and `remote` command need an `outputPath` to write out the typeCart files to. What can I change the name so it's clear?

Reply:
Why is the output path an argument of these options and not an option by itself?

Reply:
Now that we use verbs, no longer issue